conf/messages

```diff
@@ -1,2 +1,76 @@
-service.name = si-protected-user-list-admin-frontend
-service.text = This is your new service
+
```

Comment:
I'd be inclined to leave service.name in. It might be used as a default value somewhere.